contrib/opencensus-exporter-ocagent-trace/opencensus/ext/ocagent/trace_exporter/__init__.py

```diff
@@ -21,14 +21,14 @@
 
 from opencensus.common.transports import sync
 from opencensus.common.version import __version__
-from opencensus.trace.exporters import base
-from opencensus.trace.exporters.gen.opencensus.agent.common.v1 \
+from opencensus.ext.ocagent.trace_exporter import utils
```

Comment:
Ideally we shouldn't have to modify these generated files.
I think we might want to put the generated files as a separate package/repo (e.g. `opencensus-protocol`), and reference it from `opencensus-exporter-ocagent-trace`.

Reply:
These generated files are specific to the agent though right? I think it's fine to leave them in the agent package.

For shared generated files, as far as integrations and exporters are concerned, what's the difference between leaving them in the core library and packaging them up separately?

Reply:
Packaging them separately gives us a chance to publish the generated package automatically whenever we have a new protocol version, for example:
1. You make a change in `opencensus-proto` repo.
2. The `opencensus-protocol` package got build automatically and published to PyPI.
3. Existing ocagent exporter will not be affected since it has a strict dependency on the previous version of `opencensus-protocol` package.
4. If you're ready to move forward, update the ocagent exporter code and bump `opencensus-protocol` dependency version.

In this way we don't have to manually generate/copy files. It also makes the repo a bit smaller.